war/src/main/webapp/css/style.css

```diff
@@ -1897,6 +1903,7 @@ body.no-sticker #bottom-sticker {
 .icon-sm {
     width: 16px;
     height: 16px;
+    float: left;
```

Comment:
This causes a weird UI in one example:
<img width="1156" alt="Screen Shot 2019-04-08 at 4 35 03 PM" src="https://user-images.githubusercontent.com/791275/55758410-51d4de80-5a1c-11e9-9112-5638a6feadcb.png">

Perhaps some flex-box layout would work better? Not really that experienced, so I can't say for sure myself.

Reply:
Why is this particular change needed? What does it have to do with splitting long words?